### ISSUE TITLE
"await" is not defined

### ISSUE THREAD

**Issue by stephanos** [babel/babel-eslint#82]
I get this from babel-eslint:

```
20:6  error  "await" is not defined  no-undef
```

I'm using Babel's `asyncToGenerator` for async/await. 

I don't want to disable `no-undef` entirely. Is there any way to disable the rule *just* for 'await'?

**Comment by sebmck** [comment]
Unsure why you're getting this. Can you provide the code that this is triggered on?

**Comment by stephanos** [comment]
Ah, shoot. I used `await` in a function without `async` - my bad :)
